### ISSUE TITLE
Manually updating individual title causes Start Reading button to ignore blocked list

### ISSUE THREAD

**Issue by Mnotic** [nekomangaorg/Neko#1974]
### Steps to reproduce

1. Block group
2. Manually update individual title with blocked group(includes merging)
3. Begin reading via Start Reading button in library

### Expected behavior

Start Reading should open unblocked chapter

### Actual behavior

Start Reading opens blocked chapter

### Neko version

2.18.0

### Android version

14

### Device

Samsung Galaxy S24 Ultra

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [ ] I have written a short but informative title.
- [X] I have updated the app to the newest version **[Latest](https://github.com/nekomangaorg/Neko/releases/latest)**.
- [X] I have filled out all of the requested information in this form.

**Comment by nonproto** [comment]
How are you manually updating the individual title?

**Comment by Mnotic** [comment]
Pull down on chapter view of series

https://github.com/user-attachments/assets/cc578c1e-cfeb-40ca-b3a2-12197dbe7c15

Doing a library update will make the button ignore blocked groups again
